Source/Scene/Cesium3DTileContentFactory.js

```diff
@@ -118,20 +119,28 @@ const Cesium3DTileContentFactory = {
     const dataView = new DataView(arrayBuffer, byteOffset);
     const byteLength = dataView.getUint32(8, true);
     const glb = new Uint8Array(arrayBuffer, byteOffset, byteLength);
-    return ModelExperimental3DTileContent.fromGltf(
-      tileset,
-      tile,
-      resource,
-      glb
-    );
+    //// PROPELLER HACK  rolled back
+    if (tileset.enableModelExperimental) {
```

Comment:
Is `tileset` the parsed JSON?  If so, `extras` might be a better place to do this.  https://github.com/CesiumGS/3d-tiles/tree/main/specification#extras

![image](https://user-images.githubusercontent.com/484870/158080718-7652d90a-11a7-49c5-bab6-a13d3ca52aee.png)

Reply:
@katSchmid has pointed out that this is straight from the previous Cesium version https://github.com/CesiumGS/cesium/blob/1.90/Source/Scene/Cesium3DTileContentFactory.js#L20